### PR TITLE
Make some filesystemcontentstore traces optional.

### PR DESCRIPTION
**Pull request by SergeyTeplyakov**
This PR makes some of the messages from `FileSystemContentStoreInternal` optional (and disabled by default).

For instance, hashing, place file and hard link creation are responsible for about 8% of overall kusto traffic. And now these messages are still available but only when the configuration turns on.

### CODE REVIEW

File: Public/Src/Cache/ContentStore/Library/Tracing/EvictResult.cs [hunk 1]
```diff
@@ -116,11 +116,17 @@ public DeleteResult ToDeleteResult(ContentHash contentHash)
...
+
+        /// <nodoc />
+        public TimeSpan? EffectiveAge => CreationTime - EffectiveLastAccessTime;
+
         /// <inheritdoc />
         public override string ToString()
         {
             return Succeeded
-                ? $"Success Size={EvictedSize} Files={EvictedFiles} Pinned={PinnedSize} LastAccessTime={LastAccessTime} Age={CreationTime - LastAccessTime} ReplicaCount={ReplicaCount} EffectiveLastAccessTime={EffectiveLastAccessTime} EffectiveAge={CreationTime - EffectiveLastAccessTime}"
+                ? $"Success Size={EvictedSize} Files={EvictedFiles} Pinned={PinnedSize} LastAccessTime={LastAccessTime} Age={Age} ReplicaCount={ReplicaCount} EffectiveLastAccessTime={EffectiveLastAccessTime} EffectiveAge={EffectiveAge}"
```
Comment (JuanCarlosGI) [inline Public/Src/Cache/ContentStore/Library/Tracing/EvictResult.cs]: >LastAccessTime={LastAccessTime} [](start = 88, length = 31)

Weren't we going to remove LastAccessTime and EffectiveLastAccessTime, in favor of just having Age and EffectiveAge?

Comment (SergeyTeplyakov) [reply Public/Src/Cache/ContentStore/Library/Tracing/EvictResult.cs]: We trace the eviction manually instead of using `Result.ToString`.

**Pull request merged**